mlrun/projects/project.py

```diff
@@ -1847,11 +1847,14 @@ def set_secrets(
                 "must specify secrets OR file_path"
             )
         if file_path:
-            secrets = dotenv.dotenv_values(file_path)
-            if None in secrets.values():
-                raise mlrun.errors.MLRunInvalidArgumentError(
-                    "env file lines must be in the form key=value"
-                )
+            if path.exists(file_path):
+                secrets = dotenv.dotenv_values(file_path)
+                if None in secrets.values():
+                    raise mlrun.errors.MLRunInvalidArgumentError(
+                        "env file lines must be in the form key=value"
+                    )
+            else:
+                raise mlrun.errors.MLRunNotFoundError(f"{file_path} does not exist.")
```

Comment:
```suggestion
                raise mlrun.errors.MLRunNotFoundError(f"{file_path} does not exist")
```
(it's the convention in the code base)